addons/sale/models/sale.py

```diff
@@ -274,6 +274,7 @@ def _compute_expected_date(self):
         """ For service and consumable, we only take the min dates. This method is extended in sale_stock to
             take the picking_policy of SO into account.
         """
+        self.mapped("order_line")  # Prefetch indication
```

Comment:
why it's not 
```suggestion
        self.read("order_line")  # Prefetch indication
```

?

Reply:
I tested it, but it didn't work.

What I want in reality is to let Odoo know which `sale.order.line` records are involved, so further loops don't have to prefetch again.

Reply:
ok, the same is used in website

https://github.com/odoo/odoo/blob/013efbdfbf6bd6513a0c834ad61b807c780c5e5d/addons/website/models/website.py#L155